Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -101,15 +101,26 @@
       params:
         volume: -6
   - type: ItemToggleMeleeWeapon
+    activatedSoundOnHit:
```

Comment:
everything you see in green needs Delta-V Comments

everything in red needs to be commented out instead of deleted, and Delta-V comments added

Reply:
I have added comments onto every changed value. Should I move the audio file to DeltaV audio resources too?